chainerio/cache/file_cache.py

```diff
@@ -77,21 +77,33 @@ def unlock(self):
 
 
 class FileCache(cache.Cache):
-    '''Stores cache data in local filesystem
+    '''Cache system with local filesystem
 
-    Stores cache data in local filesystem,
+    Stores cached data in local filesystem,
     ``~/.chainer/chainerio/cache`` by default. Cache data is
-    automatically deleted after the object is collected. If the
-    process quit (typically by SIGTERM) the cache remains after the
-    death of process.
+    automatically deleted after the object is collected. When this
+    object is not correctly closed, (e.g., the process killed by
+    SIGTERM), the cache remains after the death of process.
+
+    Arguments:
+        length (int): Length of the cache array.
+        multithread_safe (bool): Defines multithread safety. If this
+            is ``True``, reader-writer locking is introduced behind
+            the cache system. Major use case is with Chainer's
+            ``MultithreadIterator``.
+        do_pickle (bool):
+            Do automatic pickle and unpickle inside the cache.
+        dir (str):
+            The path to directory to place cache data in case home
```

Comment:
https://github.com/chainer/chainerio/pull/63#discussion_r327915198

Reply:
I didn't get what you meant. Could elaborate a bit more?

Reply:
Sorry to make it confusing. It was a suggestion, to change the description of "dir" to

```
The directory where the cache data will be stored. In the default case, the home directory is used. Recommend to set the `dir` to a fast storage device and avoid using remote filesystems, such NFS.
```

Reply:
I think the second sentence is broken, and I think the change is trivial.